.env.sample

```diff
@@ -89,3 +89,6 @@ FTP_HOST=
 FTP_USER_NAME=
 FTP_USER_PASSWORD=
 FTP_PORT=
+
+# Project configs directory
+PROJECT_CONFIGS_DIR=
```

Comment:
We're having fun changing the env file a fair bit. Would you be able to write a pixl CLI function which reads in the `.env` and the `.env.sample`, and logs with warning that we havent added a key into the `.env`? 

To get decouple to read in a specific .env file you can use:
```
from decouple import Config, RepositoryEnv

env_config = Config(RepositoryEnv(".env.sample"))
```

Reply:
Would it make more sense as a test that runs with github actions? Maybe part of system tests?

Reply:
Ah I'm more concerned about an existing deployment on a GAE not updating its `.env` after the `.env.sample` has been updated

Reply:
Done, if I understood you correctly - I'm not super happy with the implementation (loop of try-catch), but couldn't find a better way, and it should mostly not fail so :shrug: